services/ui-src/src/components/tables/EntityRow.tsx

```diff
@@ -0,0 +1,116 @@
+// components
+import { Button, Image, Td, Tr } from "@chakra-ui/react";
+// utils
+import { AnyObject, EntityShape } from "types";
+// assets
+import deleteIcon from "assets/icons/icon_cancel_x_circle.png";
+import unfinishedIcon from "assets/icons/icon_error_circle.png";
+
+export const EntityRow = ({
+  entity,
+  verbiage,
+  openAddEditEntityModal,
+  openDeleteEntityModal,
+}: Props) => {
+  const { programName, planName } = entity;
+
+  const reportingPeriod = `${entity.reportingPeriodStartDate} to ${entity.reportingPeriodEndDate}`;
+  const eligibilityGroup = () => {
+    if (entity["eligibilityGroup-otherText"]) {
+      return entity["eligibilityGroup-otherText"];
+    }
+    return entity.eligibilityGroup[0].value;
+  };
+
+  const programInfo = [
+    programName,
+    eligibilityGroup(),
+    reportingPeriod,
+    planName,
+  ];
+
+  return (
+    <Tr sx={sx.content}>
+      <Td sx={sx.iconColumn}>
+        <Image src={unfinishedIcon} alt="warning icon" boxSize="xl" />
+      </Td>
+      <Td sx={sx.programInfo}>
+        <ul>
+          {programInfo.map((field, index) => (
+            <li key={index}>{field}</li>
+          ))}
+        </ul>
+      </Td>
+      <Td>
+        {openAddEditEntityModal && (
+          <Button
+            variant="none"
+            sx={sx.editButton}
+            onClick={() => openAddEditEntityModal(entity)}
+          >
+            {verbiage.editEntityButtonText}
+          </Button>
+        )}
+      </Td>
+      <Td>
+        {/* TODO: Enter MLR report routing */}
```

Comment:
Do we still need this TODO?

Reply:
I think it's reasonable to leave it—this button needs logic to navigate to the MLR report overlay